test/functional/wallet_bumpfee.py

```diff
@@ -305,6 +306,27 @@ def test_locked_wallet_fails(rbf_node, dest_address):
                             rbf_node.bumpfee, rbfid)
     rbf_node.walletpassphrase(WALLET_PASSPHRASE, WALLET_PASSPHRASE_TIMEOUT)
 
+def test_change_script_match(rbf_node, dest_address):
+    rbfid = spend_one_input(rbf_node, dest_address)
+    unbumped_details = rbf_node.getrawtransaction(rbfid, 1)
+    bumped_total_tx = rbf_node.bumpfee(rbfid, {"totalFee": 2000})
+    total_bumped_details = rbf_node.getrawtransaction(bumped_total_tx["txid"], 1)
+    bumped_rate_tx = rbf_node.bumpfee(bumped_total_tx["txid"])
+    rate_bumped_details = rbf_node.getrawtransaction(bumped_rate_tx["txid"], 1)
+
+    # find change from original
+    for out in unbumped_details["vout"]:
+        address = out["scriptPubKey"]["addresses"][0]
+        if rbf_node.getaddressinfo(address)["ischange"] == True:
+            break
+    assert(address is not None)
```

Comment:
In commit "wallet_bumpfee.py: add test for change key preservation" (5cedf2295934637af451e977ed13d61762f41d0b)

If none of the outputs are change, `address` will just be set to the last output address. Probably should add `change_address = None` before the loop, `change_address = address` before the break, and use `change_address` instead of `address` below.